### PR TITLE
Michael kennedy39 castleton

### PR DESCRIPTION
**Pull request by MichaelKennedy39**
Add MichaelKennedy39 hometown Castleton Vt

### CODE REVIEW

File: MichaelKennedy39-Castleton.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Castleton, VT
+### Restaurants
+Rinquists
```
Comment (ShaunHaageCLAB) [inline MichaelKennedy39-Castleton.md]: @MichaelKennedy39 is this place any good?

**Comment by MichaelKennedy39** [comment]
Have you ever been to Castleton ?

File: MichaelKennedy39-Castleton.md (hunk 1 above)
Comment (MichaelKennedy39) [inline MichaelKennedy39-Castleton.md]: Great food !

Comment (ShaunHaageCLAB) [inline MichaelKennedy39-Castleton.md]: awesome!

**Comment by MichaelKennedy39** [comment]
Am I good to commit ?

**Pull request merged**